quarkchain/evm/specials.py

```diff
@@ -354,10 +366,12 @@ def addr(self) -> bytes:
     SystemContract.ROOT_CHAIN_POSW: (
         decode_hex(b"514b430000000000000000000000000000000001"),
         ROOT_CHAIN_POSW_CONTRACT_BYTECODE,
+        0,
     ),
     SystemContract.NON_RESERVED_NATIVE_TOKEN: (
         decode_hex(b"514b430000000000000000000000000000000002"),
         NON_RESERVED_NATIVE_TOKEN_CONTRACT_BYTECODE,
+        0,
```

Comment:
as discussed offline, this will cause a hard fork if `ENABLE_MNT_AUCTION_TIMESTAMP` is not set